cg/apps/balsamic/api.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from cg.utils.commands import Process
+from pathlib import Path
+
+LOG = logging.getLogger(__name__)
+
+
+class BalsamicAPI:
+    """Handles execution of BALSAMIC"""
+
+    EXIT_SUCCESS = 0
+
+    def __init__(self, config):
+        self.binary = config["balsamic"]["binary_path"]
+        self.singularity = config["balsamic"]["singularity"]
+        self.reference_config = config["balsamic"]["reference_config"]
+        self.root_dir = config["balsamic"]["root"]
+        self.account = config["balsamic"]["slurm"]["account"]
+        self.email = config["balsamic"]["slurm"]["mail_user"]
+        self.qos = config["balsamic"]["slurm"]["qos"]
+        self.bed_path = config["bed_path"]
+        self.process = Process(self.binary)
+
+    def __build_command_str(self, options) -> list:
```

Comment:
```suggestion
    def __build_command_str(self, options: dict) -> List[str]:
```

Reply:
Also I wonder, is there a typo that there are two underscores in the function name? Convention is to use one underscore for private functions and two before and after for magic functions such as `__init__` and `__iter__`